.buildkite/pipeline.yml

```diff
@@ -53,7 +53,6 @@ steps:
         matrix:
           setup:
             ruby:
-              - 2.7.4
```

Comment:
[The trunk branch protection rule](https://github.com/wordpress-mobile/release-toolkit/settings/branch_protection_rules/) needs to be changed to accommodate the pipeline change.

Reply:
Yep, I've seen there's this required check that is pending, but was removed.

I don't seem to have access to the branch protection rules in `release-toolkit` though 🤔 . In fact, I don't see the Settings tab at all. Perhaps you or someone else can change it / change my role in the repo?

Reply:
Done!